acceptance/tests/resource/service/windows.rb

```diff
@@ -172,8 +219,9 @@ def service_manifest(name, params)
         assert_match(/#{mock_service_long_start_stop[:name]}/, result.stderr, 'No progress made on service operation and dwWaitHint exceeded')
       end
     end
+  end
 
-    # delete the service so it doesn't interfere with subsequent tests
+  teardown do
```

Comment:
should register the teardown at the beginning of the test, as if it fails in one of the previous steps this block won't be executed